Annotation scripts guide (#3)
ome/omero-guide-scripts: docs/extended_description.rst

under "*Advanced parameters*".

.. csv-table::
   :header: "OBJECT_NAME", "OBJECT_ID", "tag"
   :widths: 20, 10, 20

   "A.tif", "120", "Tag_1, Tag_2, Tag_3"
   "B.jpg", "121", "Tag_1[*Tag-Set*]"
   "C.jpg","122","Tag_1[*Tag-Id*]"

Choosing the CSV separator
--------------------------
When importing annotations from a .csv file, the script tries by default \
to detect the CSV separator automatically (one of ``,`` ``;`` ``TAB``).

It is possible to specify directly which one is used (in the case the automatic \
detection fails for example). As the text in the annotations may contain \
commas or semi-column, it is recommended to use ``TAB`` as separators.

Columns of parent names
-----------------------
To clarify the provenance of objects listed in the csv, you can choose to export \
additional columns for the name of the parent objects.

Those columns are by default excluded from the Key-value pairs using the import \
script (<PARENT> value of the "Columns to exclude" parameter matches all parent \
containers: PROJECT, DATASET, SCREEN, PLATE, WELL and RUN)

Default Namespace
-----------------

> Leaving the namespace parameter to blank always refers to the same namespace, \

Reviewer: ```suggestion
Leaving the namespace parameter to blank is the same as refering to \
```

> the "Client namespace", corresponding to the one given to new Key-Value pairs \

Reviewer: ```suggestion
the "Client namespace" (``openmicroscopy.org/omero/client/mapAnnotation``). The "Client namespace" \
is used when a new Key-Value pair is created in OMERO.web user interface. \
```

> created inside OMERO.web. This namespace \

Reviewer: ```suggestion
Only Key-Value pairs with this "Client namespace" are editable in OMERO.web user interface. \
```

> (``openmicroscopy.org/omero/client/mapAnnotation`` in full) is treated \

Reviewer: ```suggestion

```

> differently by OMERO.web as it is the only one that can be edited in its \

Reviewer: ```suggestion

```

> interface.

Reviewer: ```suggestion

```
